Rakefile

```diff
@@ -57,6 +57,43 @@ def format_links(text)
   text.gsub(/{([^}]+)}/, '[`\1`](#\1)')
 end
 
+namespace :travis do
+  task rubocop: :rubocop
+  task :unit do
+    sh "docker-compose -f spec/docker-compose.yml build --parallel ubuntu_node puppet_5_node puppet_6_node"
+    sh "docker-compose -f spec/docker-compose.yml up -d ubuntu_node puppet_5_node puppet_6_node"
+    sh "r10k puppetfile install"
+    Rake::Task['travisci'].invoke
+  end
+  task :modules do
+    success = true
+    %w[boltlib ctrl file out system].each do |mod|
+      Dir.chdir("#{__dir__}/bolt-modules/#{mod}") do
+        sh 'rake spec' do |ok, _|
+          success = false unless ok
+        end
+      end
+    end
+    raise "Module tests failed" unless success
+  end
+  task docs: :generate_docs
+  task :integration do
+    sh "docker-compose -f spec/docker-compose.yml build --parallel"
```

Comment:
👍 on `--parallel`. I never realized that existed!

I *believe* the reason that I built `samba-ad` first was because it needs to be up and running for the Travis CI runner to be able to connect as a Kerberos client - you can look at the commit here - https://github.com/puppetlabs/bolt/commit/4637ffc8ff4e2443d9fc13a9f5de531965b6de9e#diff-354f30a63fb0907d4ad57269548329e3R30 for when that was originally introduced.

I don't think you'll need to update docs for any changes you've made here, but you might want to double check https://github.com/puppetlabs/bolt/blob/master/developer-docs/kerberos.md